css/styles.css

```diff
@@ -400,3 +409,129 @@ button,
 .d-grid {
   display: grid;
 }
+
+@media screen and (min-width: 992px) {
+  /* Body limited to 1100px */
+
+  html {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  body {
+    max-width: 1100px;
+  }
+
+  .body {
+    min-width: 480px;
+  }
+
+  /* Header */
+
+  .navbar {
+    display: none;
+  }
+
+  .navbar-desktop {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .navbar-desktop ul {
+    gap: 40px;
+  }
+
+  .headline {
+    background-image: url(../img/background/headerbg.svg);
+    margin-bottom: 80px;
+    background-repeat: no-repeat;
+  }
+
+  .headline-content {
+    max-width: 700px;
+    margin: 180px 120px 150px 120px;
+    gap: 20px;
+  }
+
+  .headline-content h2 {
+    margin-top: 10px;
+  }
+
+  /* Portfolio - Cards */
+
+  .work-card {
+    display: grid;
+    grid-template-columns: auto auto;
+    grid-template-rows: auto;
+    gap: 50px;
+  }
+
+  #work-2 .card-info,
+  #work-4 .card-info {
+    grid-column-start: 1;
+    grid-row-start: 1;
+  }
+
+  .my-works img {
+    max-width: 480px;
+  }
+
+  /* About Section */
+
+  .about-myself {
+    display: grid;
+    grid-template-columns: 1.5fr 2fr;
+    margin: 100px 0 120px 0;
+    gap: 50px;
+  }
+
+  .about-content {
+    margin-left: 50px;
+  }
+
+  #languages-list {
+    flex-direction: row;
+  }
+
+  .language {
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    width: 50px;
+    height: 50px;
+    padding: 30px;
+  }
+
+  .language img {
+    max-width: 100%;
+  }
+
+  .language p {
+    margin: 0;
+  }
+
+  /* Contact Me section */
+
+  footer {
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background-image: url(../img/background/footerbgdesk.svg);
+    background-repeat: no-repeat;
+    background-size: 100%;
+  }
+
+  .section-form {
+    align-items: center;
+    justify-content: center;
+    max-width: 350px;
+    padding: 50px;
+    padding-bottom: 70px;
+  }
```

Comment:
- [ ] The Contact section is must fulfill the screen. In this design, I can see some gaps at the edge of it. I highly recommend to you fulfill this gap as well. 


![Screen Shot 2021-12-01 at 3 18 03 PM](https://user-images.githubusercontent.com/69011963/144218485-d2648d14-9f74-48a3-b3c3-f2d48a462c5e.png)

Reply:
@Rustamxon7 Thank you! Will do it ASAP. 😃 .

PS: The whole website (desktop version) is limited to 1100px since in the requirements of the project they told us the screen was up to 1100px.  That's why the contact form doesn't fulfill the screen, because I'm using a fictional screen of 1100px. Actually what I'm doing here is using a max-width of 1100px but I will remove it now 👍  Thank you! 😄

Reply:
Hi @orozCoding, please fix this issue recommended by the previous reviewer also the about section and the contact section must fill the screen. please fix this.